README.md

```diff
@@ -40,15 +52,91 @@ or
 python setup.py develop
 ```
 
+## Getting started
+`s2spy` provides end-to-end solutions for machine learning (ML) based S2S forecasting.
+
+![workflow](./docs/assets/images/workflow.png)
+
+### Datetime operations & Data processing
+In a typical ML-based S2S project, the first step is always data processing.  A calendar-based datetime operation module is implemented to help the users prepare their data. For instance, if a user is looking for predictors for winter climate at seasonal timescales (~180 days), a `AdventCalendar` can be used to prepare the data:
+
+```py
+calendar = s2spy.time.AdventCalendar(anchor=(11, 30), freq='180d')
+calendar = calendar.map_years(2020, 2021)
+calendar
+>>>    i_interval                 (target) 0                         1
+>>>    anchor_year
+>>>    2021         (2021-06-03, 2021-11-30]  (2020-12-05, 2021-06-03]
+>>>    2020         (2020-06-03, 2020-11-30]  (2019-12-06, 2020-06-03]
+```
+
+`s2spy` works with `xarray` data. Now, the user can load their data and easily resample their data to the desired timescales configured in the calendar:
+
+```py
+calendar = calendar.map_to_data(input_data)
+bins = s2spy.time.resample(calendar, input_data)
+>>>       anchor_year  i_interval                  interval  mean_data  target
+>>>     0        2020           0  (2020-06-03, 2020-11-30]      275.5    True
+>>>     1        2020           1  (2019-12-06, 2020-06-03]       95.5   False
+>>>     2        2021           0  (2021-06-03, 2021-11-30]      640.5    True
+>>>     3        2021           1  (2020-12-05, 2021-06-03]      460.5   False
+```
+
+Depending on their tasks, the user can choose their desired calendar from a collection of different types of calendars, to process their data (e.g. `MonthlyCalendar` and `WeeklyCalendar`).
+
+### Cross-validation
+Using `s2spy`, users can easily perform cross-validation by calling a splitter and generating train/test splits with the resampled data:
+
+```py
+bins = s2spy.time.resample(calendar, input_data)
+
+from sklearn.model_selection import ShuffleSplit
+splitter = ShuffleSplit(n_splits=3)
+s2spy.traintest.split_groups(splitter, bins)
+```
+
+All splitter classes from `scikit-learn` are supported, a list is available [here](https://scikit-learn.org/stable/modules/classes.html#splitter-classes).
```

Comment:
perhaps, we can add a sentence saying that users should follow scikit-learn documentation on how to use a different splitter class.

Reply:
Good advice! I add it to the readme.